Project.toml

```diff
@@ -1,15 +1,13 @@
 name = "LibSpatialIndex"
 uuid = "f19c2e90-9d16-5f2d-a2a7-af3fb29e4907"
 license = "MIT"
-version = "0.2.0"
+version = "0.3.0"
 
 [deps]
-BinaryProvider = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
-Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
+LibSpatialIndex_jll = "00e98e2a-4326-5239-88cb-15dcbe1c18d0"
 
 [compat]
-BinaryProvider = "0.5"
-julia = "1"
+julia = "1.3"
```

Comment:
Here we also need a compat entry for LibSpatialIndex_jll, to make sure that it can be automatically merged in the registry.

Reply:
I set `LibSpatialIndex_jll = "1.8"`